thumbor/error_handlers/sentry.py

```diff
@@ -1,87 +1,53 @@
-#!/usr/bin/python
-# -*- coding: utf-8 -*-
-
-# thumbor imaging service
-# https://github.com/thumbor/thumbor/wiki
-
-# Licensed under the MIT license:
-# http://www.opensource.org/licenses/mit-license
-# Copyright (c) 2011 globo.com thumbor@googlegroups.com
-
-import pkgutil
-import pkg_resources
+import sentry_sdk
+from sentry_sdk.integrations.logging import ignore_logger
+from sentry_sdk.utils import event_from_exception, exc_info_from_error
 
 from thumbor import __version__
 
-try:
-    basestring        # Python 2
-except NameError:
-    basestring = str  # Python 3
-
 
 class ErrorHandler(object):
-    def __init__(self, config, client=None):
-        import raven
+    def __init__(self, config):
+        self.config = config
 
         dsn = config.SENTRY_DSN_URL
+
         if not dsn:
             raise RuntimeError(
-                "If you set USE_CUSTOM_ERROR_HANDLING to True, and you are using thumbor.error_handlers.sentry, " +
+                "If you set USE_CUSTOM_ERROR_HANDLING to True, and you are using thumbor.error_handlers.sentry_python, " +
```

Comment:
I think that has to stay as it was.

Reply:
Yes, you're right. I've pushed a commit to change it back.